pkg/services/serviceaccounts/api/token.go

```diff
@@ -246,7 +246,7 @@ type DeleteTokenParams struct {
 // swagger:response listTokensResponse
 type ListTokensResponse struct {
 	// in:body
-	Body *TokenDTO
+	Body []*TokenDTO
```

Comment:
Good job spotting this. I believe:
```suggestion
	Body []TokenDTO
```

Reply:
Hmm, I looked at some other responses and they had pointers as well.

Also, I generated the OpenAPI client from my PR and it worked well

So, I'm not sure. I personally would lean towards the pointer version

Reply:
Alright I wonder the reasoning behind it. It does look a bit strange to me but in the end the json does not return pointers anyway :shrug: 

Should the result be aligned then?
https://github.com/grafana/grafana/blob/ff96532dd06085e864983f20bd7cae4f0c1e5ad4/pkg/services/serviceaccounts/api/token.go#L81

Reply:
I removed the pointer. Makes no difference in Swagger, so let's use that!